functional_tests/scylla_operator/test_functional.py

```diff
@@ -693,6 +693,7 @@ def test_default_dns_policy(db_cluster: ScyllaPodCluster):
         f"Pods: {yaml.safe_dump(pods_with_wrong_dns_policy, indent=2)}")
 
 
+@pytest.mark.skip("Disabled due to the https://github.com/scylladb/scylla-operator/issues/1077")
```

Comment:
it still annoys me a bit we'll loss the viability of this one on EKS weekly run

I wish we could use something like:
https://docs.pytest.org/en/7.1.x/reference/reference.html#_pytest.hookspec.pytest_markeval_namespace

so we could use them inside `pytest.mark.skipIf`

but it's limit to any information we can get before any fixture is being called.

Reply:
Done